### DISCUSSION TITLE
Display play count and other stats on the leaderboard and beatmap

### DISCUSSION THREAD

**Discussion by Stehsaer** [ppy/osu#25929] [Ideas] [answered]
- Display one's play count of a player on the leaderboard in one specific beatmap. For example, one who plays a beatmap for 120 times and gets the first place on the leaderboard, given that it may be difficult to stuff into compact display, I suggest displaying their play count in the tooltip shown when hovered.
- Display the play count of a beatmap of the player themselves. For example, I played Cycle hit for 100 times, I suggest displaying my own play count on the left of the song select page
- All these suggestion above may also apply to osu!web

**Comment by peppy** [comment, marked as answer]
Completely agree
